test/EjectCommand.js

```diff
@@ -0,0 +1,78 @@
+import assert from "assert";
+import exitWithCode from "./_exitWithCode";
+import initFixture from "./_initFixture";
+import EjectCommand from "../src/commands/EjectCommand";
+import logger from "../src/logger";
+import stub from "./_stub";
+
+describe("EjectCommand", () => {
+
+  describe("in a basic repo", () => {
+    beforeEach((done) => {
+      initFixture("EjectCommand/basic", done);
+    });
+
+    it("should eject a single package", (done) => {
+      const pkg = "package-1";
+      const ejectCommand = new EjectCommand([pkg], {dry: true});
+
+      ejectCommand.runValidations();
+      ejectCommand.runPreparations();
+
+      let index = 0;
+      const patterns = [
+        "The following directories would be moved",
+        pkg
+      ];
+      stub(logger, "info", (message) => {
```

Comment:
Instead of stubbing the logger and passing `dry: true`, why not stub `FileSystemUtilities.renameSync` and assert it passed the args you expect?

(when asserting multiple stubs in sequence, I've found [`assertStubbedCalls`](https://github.com/asini/asini/blob/master/test/_assertStubbedCalls.js) to be useful)